python/create_init.py

```diff
@@ -21,6 +21,8 @@
 imports.sort()
 languages.sort()
 
+assert languages
```

Comment:
Minor change to be explicit on the length check, and use an explicit assertion (to avoid `-O` impact).
```suggestion
if len(languages) == 0:
    raise AssertionError('languages list is empty!')
```

Reply:
I think `if not languages` reads well too; it's clearly saying that languages has to be something to pass. I considered avoiding -O, but this is a utility script so I thought that was unlikely to be present. 

I'll change to match but personally prefer `if not languages`.

Reply:
Happy with whatever you think best.